packages/cascara/src/ui/Columns/Columns.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import pt from 'prop-types';
+import { Role } from 'reakit';
+import classNames from 'classnames/bind';
+
+import Boundaries from '../../system-components/Boundaries';
+import styles from './Columns.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /** The section content */
+  children: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  /** Columns can have css class name */
+  className: pt.string,
+  /** The number of columns to allocate */
+  count: pt.number,
+};
+
+const Columns = ({ count = 2, children, className, ...rest }) => (
+  <Boundaries>
+    <Role
+      className={cx('Columns', className)}
+      style={{ columnCount: count }}
```

Comment:
Manu, for now let's not change this... but we may need to consider the possibility that we allow direct, inline styles in some cases if someone needs to fix something. In that scenario, we might need to have a solution to merge objects if a user defines a `style` prop. In the current scenario, technically that will not work and we will throw any of their styles away.

That can be discussed separately, but we should be aware of this. We might be giving a weird experience to users as some places they can use `style` where we do a rest spread, and others, like this, they cannot.

Reply:
Let's not make a change here, but we are going to need a style merge helper function to make this easier to do.

Reply:
@brianespinosa I did not create the helper function, as you said, we need to discuss it separately. But the component now accepts an extra prop `style` that is merged with the internal inline style. 👍